cobbler/settings/__init__.py

```diff
@@ -367,24 +369,39 @@ def __getattr__(self, name):
                     f"no settings attribute named '{name}' found"
                 ) from error
 
-    def save(self, filepath="/etc/cobbler/settings.yaml"):
+    def save(self, filepath="/etc/cobbler/settings.yaml", ignore_keys=None):
```

Comment:
```suggestion
    def save(self, filepath="/etc/cobbler/settings.yaml", ignore_keys: Optional[List[str]] = None):
```

let's give those a type annotation to state what this is about (please also adjust the rest of the arguments).